_IBC/milkyway-babylon.json

```diff
@@ -0,0 +1,31 @@
+{
+  "$schema": "../ibc_data.schema.json",
+  "chain_1": {
+    "chain_name": "milkyway",
```

Comment:
wrong chain order. Must be alphabetical for both chain_1 chain_2 and also the file name, so (babylon-milkyway, not milkyway-babylon)

Reply:
Thank you, I just realized that I can run the validation scripts locally, too. I updated the file name and confirmed that validations are now passing.